src/VisualStudio/Core/Def/ProjectSystem/CPS/IWorkspaceProjectContextFactory.cs

```diff
@@ -70,17 +75,29 @@ internal abstract class EvaluationData
         /// <returns>
         /// Returns empty string if the property is not set.
         /// </returns>
-        /// <exception cref="ArgumentException">
+        /// <exception cref="InvalidProjectDataException">
         /// The <paramref name="name"/> is not listed in <see cref="IWorkspaceProjectContextFactory.EvaluationPropertyNames"/>
         /// </exception>
         public abstract string GetPropertyValue(string name);
 
+        /// <summary>
+        /// Returns the values of items of the specified <paramref name="name"/>.
+        /// </summary>
+        /// <returns>
+        /// Returns empty array if the items are not set.
+        /// </returns>
+        /// <exception cref="InvalidProjectDataException">
+        /// The <paramref name="name"/> is not listed in <see cref="IWorkspaceProjectContextFactory.EvaluationItemNames"/>
+        /// </exception>
+        public virtual ImmutableArray<string> GetItemValues(string name)
```

Comment:
"Values" seems like an odd suffix on this method name, just GetItems? Do items have a value versus the item spec vs. something else?

Reply:
Yes, values here mean item specs. We might want to add `GetItems` in future if we need to read item metadata. For the purpose of this API items are like properties with multiple values.